concepts/src/concept_csv_export.py

```diff
@@ -24,6 +24,8 @@
 VERSION = 2.3
 LOCALES = ['en']
 DEFAULT_LOCALE = 'en'
+ENCODING = 'latin-1'
```

Comment:
Shouldn't this default to utf-8?

Reply:
Yeah probably, I left it as `latin-1` for backwards compatibility